catalog-info.yaml

```
@@ -35,8 +35,7 @@ spec:
   owner: group:data-analysis
   system: analysis-platform
   dependsOn:
-    - ispyb-api
-    # graph-federation when added
+    - resource: resource:ispyb-database
```

This doesn't look like the other two examples. Is it supposed to just be
```suggestion
    - resource:ispyb-database
```
?